<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoinUI</class>
 <widget class="QDialog" name="JoinUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="uiListColTab">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="uiListColAv">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="uiButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="uiLabelFirstCol">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Coloana Selectata:</string>
   </property>
  </widget>
  <widget class="QLabel" name="uiLabelSecondCol">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Coloana Selectata:</string>
   </property>
  </widget>
  <widget class="QLabel" name="uiLabelMesaj">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Inner Join</string>
   </property>
  </widget>
  <widget class="QLabel" name="uiLabelFirstTabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Coloanele tabelei:</string>
   </property>
  </widget>
  <widget class="QLabel" name="uiLabelColMes">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Coloanele valabile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uiButtonSelect1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uiButtonSelect2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
